readthedocs/api/v3/tests/mixins.py

```diff
@@ -43,6 +43,7 @@ def setUp(self):
         # objects (like a Project for translations/subprojects)
         self.project = fixture.get(
             Project,
+            id=1,
```

Comment:
While running apiv3 tests on .com with -k this was changed to 3, better to always be deterministic.